lib/model-builder.js

```diff
@@ -526,10 +527,13 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
 };
 
 // DataType for Date
-function DateType(arg) {
+function DateType(arg, props) {
```

Comment:
What was the reasoning behind have this function both in `dao` and `modelBuilder`?